### PR TITLE
[Feature] Added "ToMatchEcho" matcher.

### PR DESCRIPTION
**Pull request by m1ome**
New matcher to be able to match a echoed output handy.

### CODE REVIEW

**Comment by jails** [comment]
:+1: !
To avoid some code duplication could be interesting to extend it from `ToEcho` ?

**Comment by m1ome** [comment]
Ok, i'll try to do it

**Comment by m1ome** [comment]
@jails, done.

**Comment by m1ome** [comment]
If you want you can squash, i think this will be the best choice.

**Comment by jails** [comment]
:+1: !

File: docs/matchers.md [hunk 1]
```diff
@@ -204,6 +204,28 @@ it("passes if $closure echoes the expected output", function() {
 });
 ```
 
+**toMatchEcho($expected)**
```
Comment (samsonasik) [inline docs/matchers.md]: can update the `kahlan/docs` also : https://github.com/kahlan/docs/blob/master/matchers.md ?

Comment (samsonasik) [inline docs/matchers.md]: just realize it already in docs ..

**Pull request merged**